tests/fixtures/metrics_fixtures.py

```diff
@@ -23,6 +23,16 @@ def test_implicit_predicted_scores():
     ])
 
 
+@pytest.fixture()
+def test_implicit_predicted_scores_negative_sample_user():
+    return torch.tensor([
+        [0.9, 0.1, -1],
+        [0.4, 1.2, -2],
+        [0.5, 1.7, -3],
+        [0.7, 0.12, 4],
+    ])
```

Comment:
It looks like this is just `test_implicit_predicted_scores` but transposed. If that's intentional, can we just use that logic to make this fixture, rather than have it fully typed out?